src/jit/codegencommon.cpp

```diff
@@ -2349,6 +2368,25 @@ void CodeGen::genGenerateCode(void** codePtr, ULONG* nativeSizeOfCode)
 
     genSetScopeInfo();
 
+#if DEBUG
+    if (compiler->verbose)
```

Comment:
It started just for debugging purposes, then I thought it would be useful to someone else. I will leave this without resolution so I can close it when the PR is about to be merged.